python/pyspark/sql/session.py

```diff
@@ -679,13 +679,13 @@ def createDataFrame(self, data, schema=None, samplingRatio=None, verifySchema=Tr
                             "true." % _exception_message(e))
                         warnings.warn(msg)
                     else:
-                        msg = (
+                        e.message = (
```

Comment:
@BryanCutler, I think `message` attribute is only in Python 2. Also, are you doubly sure if this wraps the exception message in console too .. ?

Reply:
Ah yes, you're right.  The tests check the exception message so I thought all was good.  Let me try something else.